### ISSUE TITLE
How get blessings version?

### ISSUE THREAD

**Issue by ocastilloreyes** [erikrose/blessings#118]
I need get the blessings version but I have the following error: 

>>> import blessings
>>> blessings.__version__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute '__version__'

Thank you

**Comment by Sbelletier** [comment]
try blessings.__version__
it's a python convention

**Comment by erikrose** [comment]
There isn't a convenient way right now, but let's leave this open as a reminder to add one in a future release.

**Comment by jquast** [comment]
```python
import pkg_resources
pkg_resources.get_distribution("blessings").version
```

**Comment by erikrose** [comment]
Yes, that should work if blessings was installed by `pip` or maybe `easy_install`. Thanks, Jeff!
